pkgs/development/python-modules/nvchecker/default.nix

```
@@ -71,6 +74,13 @@ buildPythonPackage rec {
     "-m 'not needs_net'"
   ];
 
+  passthru.optional-dependencies = {
```

```suggestion
  optional-dependencies = {
```
We now accept the bare `optional-dependencies` since https://github.com/NixOS/nixpkgs/commit/b9138b7c072f55dcfd7b8c0747c3c4291bd0b28e